receiver/sapnetweaverreceiver/metadata.yaml

```diff
@@ -162,7 +165,7 @@ metrics:
       value_type: int
     attributes: []
     enabled: true
-# 10: ICM logs - cumulative count of errors: NOT IMPLEMENTED YET/ NOT SURE IF WE WILL IMPLEMENT
+# 10: ICM logs - cumulative count of errors
```

Comment:
before we merge we should remove these comment that map to HD requests

Reply:
Removed all HD comments as they are not necessary. HD had their own order sheet and I wanted to be certain not to skip any.